tests/baselines/reference/user/webpack.log

```diff
@@ -1,22 +0,0 @@
-Exit Code: 1
-Standard output:
-lib/ContextModule.js(436,34): error TS2341: Property 'moduleGraph' is private and only accessible within class 'ChunkGraph'.
```

Comment:
need to check whether this is from webpack updating to 3.8. Probably! Hopefully!

Reply:
Yes